AsyncImageLoader.Avalonia/AdvancedImage.axaml.cs

```diff
@@ -69,7 +69,7 @@ public class AdvancedImage : ContentControl {
 
             var loaderChangedObservable = LoaderProperty.Changed
                 .Where(args => args.IsEffectiveValueChange)
-                .Where((args, i) => i == 0 || args.Sender.GetValue(ShouldLoaderChangeTriggerUpdateProperty))
+                .Where((args, i) => i == 0 || (bool)args.Sender.GetValue(ShouldLoaderChangeTriggerUpdateProperty))
```

Comment:
This change will trigger possible null reference exception warning. We should suppress it via adding `!` to the end of expression